src/lib/issues.ts

```diff
@@ -0,0 +1,60 @@
+import type { HttpdClient, Issue } from "@httpd-client";
```

Comment:
I think we can move this into `Issue.svelte` as it is only used there.

Reply:
3af3c74